addons/web/static/src/js/views/list/list_controller.js

```diff
@@ -137,6 +139,41 @@ var ListController = BasicController.extend({
             this.$buttons.appendTo($node);
         }
     },
+    /**
+     * Renders (and updates) the buttons whose description are in the `header` node of the list view arch
+     * In a list, those buttons are visible when selecting some records
+     * They will be appended to the controlPanel's buttons
+     *
+     * @private
+     *
+     * @param {Object[]} [nodeArray] An array of buttons descriptors
+     * @returns {JQueryElement}
+     */
+    _renderHeaderButtons(nodeArray) {
+        if (this.$buttonsOnActiveIds) {
+            this.$buttonsOnActiveIds.remove();
+            this.$buttonsOnActiveIds = null;
+        }
+        nodeArray = nodeArray || this.headerButtons;
+        if (!nodeArray.length || !this.selectedRecords.length) {
+            return;
+        }
+        const btnClasses = 'btn-primary btn-secondary btn-link btn-success btn-info btn-warning btn-danger'.split(' ');
+        let $elms = $();
+        nodeArray.forEach(node => {
+            const $btn = viewUtils.renderButtonFromNode(node);
+            const btn = $btn[0];
+            btn.classList.add('btn');
+            if (!btnClasses.some(cls => btn.classList.contains(cls))) {
+                btn.classList.add('btn-secondary');
+            }
```

Comment:
I wouldn't do the effort not using jquery here as we're using it everywhere anyway
```js
            const btn = $btn[0];
            btn.classList.add('btn');
            if (!btnClasses.some(cls => btn.classList.contains(cls))) {
                btn.classList.add('btn-secondary');
            }
```
```js
            $btn.addClass('btn');
            if (!btnClasses.some(cls => $btn.hasClass(cls))) {
                $btn.addClass('btn-secondary');
            }
```